source/api/annex/notes/design_patterns.md

```diff
@@ -112,11 +114,64 @@ IIIF specifications define the functionality that can be expected to work and ho
 
 The intent of adopting this pattern is to enable experimentation by implementers, thereby encouraging the early adoption and validation of new (minor) versions.
 
+## 3. JSON-LD Design Patterns
+
+The representation of the information in JSON must be:
+
+  * Valid JSON and JSON-LD, following the restrictions of both specifications ;
+  * Understandable by developers without special training or documentation ;
+  * Usable in a variety of application environments to ensure the widest adoption.
+
+These design patterns have been selected in order to maximize these features in the resulting APIs.
+
+### 3.1. Use Native Data Types
+
+Native data types in JSON are used to ensure that applications do not have to do type coercion. If the value should be an integer in the model, then the JSON representation is as a JSON number.  Similarly, if the value is an ordered list of resources, then a JSON array will be used, mapping to an rdf:List.
```

Comment:
* _"a JSON number (unquoted)"_ <- add (unquoted) to make it really clear?
 * s/will be/is/ to be consistent
 * add back ticks around `rdf:List`?